File: Python/백준/Bronze/32684. 장기/장기.py
def score(l):
    return l[0]*13+l[1]*7+l[2]*5+(l[3]+l[4])*3+l[5]*2

a = list(map(int,input().split()))
b = list(map(int,input().split()))
s1 = score(a)
s2 = score(b)+1.5
if(s1>s2):
    print('cocjr0208')
else:
    print('ekwoo')

File: C#/백준/Bronze/32684. 장기/장기.cs
using System;

namespace q{
    class w{
        static int score(int[] a){
            return a[0]*13+a[1]*7+a[2]*5+(a[3]+a[4])*3+a[5]*2;
        }
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            double s1 = score(aa);
            double s2 = score(bb)+1.5;
            if(s1>s2){
                Console.WriteLine("cocjr0208");
            }
            else{
                Console.WriteLine("ekwoo");
            }
        }
    }
}